assets/javascripts/initializers/spongepowered-navbar.js.es6

```diff
@@ -107,19 +113,20 @@ export default {
           ),
           h('div.sp-logo-menu.sp-skip-handler', [
             h('ul.sp-logo-dropdown#ddleft', { key: 'sp-logo-dropdown' }, [
-              this.ulTo('https://www.spongepowered.org', 'home', 'fa-home', 'Homepage'),
-              this.ulTo('https://forums.spongepowered.org', 'forums', 'fa-comments', 'Forums'),
-              this.ulTo('https://github.com/SpongePowered', 'github', 'fa-code', 'Code'),
-              this.ulTo('https://docs.spongepowered.org', 'docs', 'fa-book', 'Docs'),
-              this.ulTo('https://jd.spongepowered.org', 'jd', 'fa-graduation-cap', 'Javadocs'),
-              this.ulTo('https://forums.spongepowered.org/c/plugins/plugin-releases', 'plugins', 'fa-plug', 'Plugins'),
-              this.ulTo('https://www.spongepowered.org/downloads', 'dl', 'fa-download', 'Downloads'),
-              this.ulTo('https://www.spongepowered.org/chat', 'chat', 'fa-comment', 'Chat'),
+              this.ulTo('https://www.spongepowered.org', 'home', 'fa-home', 'Homepage', false),
+              this.ulTo('https://forums.spongepowered.org', 'forums', 'fa-comments', 'Forums', false),
+              this.ulTo('https://github.com/SpongePowered', 'github', 'fa-code', 'Code', false),
+              this.ulTo('https://docs.spongepowered.org', 'docs', 'fa-book', 'Docs', false),
+              this.ulTo('https://jd.spongepowered.org', 'jd', 'fa-graduation-cap', 'Javadocs', false),
+              this.ulTo('https://forums.spongepowered.org/c/plugins/plugin-releases', 'plugins', 'fa-plug', 'Plugins', false),
+              this.ulTo('https://ore.spongepowered.org', 'ore', 'https://www.spongepowered.org/assets/img/icons/ore.svg', 'Ore ' + this.sup('ore-beta', 'Beta'), true),
```

Comment:
Does adding a string to the output of h actually work?

Reply:
(I think what you actually probably want is

```js
['Ore ', this.sup('ore-beta', 'Beta')]
```

Reply:
I completely agree.